### PR TITLE
Rf produce api setup

### LINKED ISSUE #37: [REFACTOR] Add linting and tests to produce API
**Is your feature request related to a problem? Please describe.**
Folder should include linting and unit tests

**Additional context**
Branch: rf-produce-api-lint(or test)

### PR DESCRIPTION
**Pull request by LadPaule**
## Description
Linting and test to be added
List any dependencies that are required for this change.

Fixes #37

## How Has This Been Tested?
No tests included yet

### CODE REVIEW

File: src/services/produce_api/produceAPI/produceAPI/urls.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
 from django.contrib import admin
 from django.urls import path, include
-from rest_framework_swagger.views import get_swagger_view
+from rest_framework.documentation import include_docs_urls
```
Comment (kmwamasali) [inline src/services/produce_api/produceAPI/produceAPI/urls.py]: remove static documentation and go back to using get_swagger_view

File: src/services/produce_api/produceAPI/produce/models.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 
 class Produce(models.Model):
-  name = models.CharField(max_length=150)
-  thumbnail = models.ImageField(null=True)
-  description = models.TextField()
-  created_date = models.DateTimeField(auto_now_add=True)
+    name = models.CharField(max_length=150)
+    thumbnail = models.ImageField(null=True)
+    description = models.TextField()
+    price = models.DecimalField(decimal_places=3, max_digits=10, default= 00.0)
```
Comment (kmwamasali) [inline src/services/produce_api/produceAPI/produce/models.py]: Price field of the produce will accept an object id not direct price information. That will be stored in another api.

Add field id to the object model

File: src/services/produce_api/produceAPI/documentation.txt [hunk 3]
```diff
@@ -0,0 +1,67 @@
+This is a simple file containing the documentation of this produce API developed in django and
```
Comment (kmwamasali) [inline src/services/produce_api/produceAPI/documentation.txt]: change this to a raml file.
Use this for guidance https://swagger.io/specification/

**Review by kmwamasali — CHANGES_REQUESTED** [review]
Work on requested changes

File: src/services/produce_api/produceAPI/produce/models.py (hunk 2 above)
Comment (LadPaule) [reply src/services/produce_api/produceAPI/produce/models.py]: I have removed this field for now

File: src/services/produce_api/produceAPI/docs.raml [hunk 4]
```diff
@@ -0,0 +1,106 @@
+#%RAML 1.0
+---
+title: e-BookMobile API
```
Comment (kmwamasali) [inline src/services/produce_api/produceAPI/docs.raml:3]: change to farmrail Produce API

File: src/services/produce_api/.vscode/settings.json [hunk 5]
```diff
@@ -1,3 +1,3 @@
 {
   "python.linting.enabled": true
-}
\ No newline at end of file
+}
```
Comment (kmwamasali) [inline src/services/produce_api/.vscode/settings.json:3]: leave new line at end of file

File: src/services/produce_api/produceAPI/docs.raml [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+                    {
+                        data:{
+                            "id": 4,
+                            "name": "cloves",
+                            "thumbnail": null,
+                            "description": "these are the leading spices",
+                        },
+                        response code: 204
+                        sucess: true
+                    }
```
Comment (kmwamasali) [inline src/services/produce_api/produceAPI/docs.raml:106]: leave new line at end of file

**Review by kmwamasali — CHANGES_REQUESTED** [review]
few small edits

**Pull request merged**